webapp/api/store.py

```diff
@@ -92,6 +98,13 @@ def process_response(self, response):
 
         return body
 
+    def get_all_snaps(self):
```

Comment:
we should probably add a `size` param to this, some stores might have 100s or 1000s.